src/options.js

```diff
@@ -43,6 +43,7 @@ const defaults = {
   flattenTernaries: false,
   breakBeforeElse: false,
   jsxBracketSameLine: false,
+  jsxAttributesIndent: false,
```

Comment:
Also, I think `jsxAttributesAlignTagName` or `jsxAttrAlignTagName` may be a better name